en/orm/associations.rst

```diff
@@ -203,6 +203,7 @@ Possible keys for hasOne association arrays include:
 - **propertyName**: The property name that should be filled with data from the associated
   table into the source table results. By default this is the underscored & singular name of
   the association so ``address`` in our example.
+- **strategy**: Defines the query strategy to use. Can use 'select' as an option.
```

Comment:
other strategy is 'join'
https://github.com/cakephp/cakephp/blob/master/src/ORM/Association/HasOne.php#L39
if i get it right its just subquery with 'select' or staight 'join' with default strategy 'join' but someone with more knowlegde should verify it : )